site/content/docs/main/start-contributing.md

```diff
@@ -1,20 +1,138 @@
-## Contributing
+# Start Contributing Guide  
```

Comment:
Very, very small nit:
The tense on this feels odd to me. Feels like it should be "New Contributor Guide", "Getting started with Contributing" or something.

Reply:
Hi Nick,
I felt "New Contributor Guide" can be interpreted in 2 ways:
- a new guide for contributors (as in new and improved)
- a guide for new contributors

"Getting Started with Contributing" title would align with "Getting Started" title for the quickstart page.

Let me know if you'd like an edit change and I'll update.

Thanks.

Reply:
I'm in favour of "Getting started with Contributing" let's be explicit !

Reply:
Sounds good. I'll make the update to "Getting Started with Contributing" title.